pkg/descheduler/strategies/pod_antiaffinity_test.go

```diff
@@ -149,12 +162,26 @@ func TestPodAntiAffinity(t *testing.T) {
 			nodes:                   []*v1.Node{node1, node3},
 			expectedEvictedPodCount: 0,
 			nodeFit:                 true,
+			testFunc:                func(podList []v1.Pod) {},
+		},
+		{
+			description:             "No pod to evicted since all pod terminating",
+			maxPodsToEvictPerNode:   0,
+			pods:                    []v1.Pod{*p9, *p10},
+			nodes:                   []*v1.Node{node1},
+			expectedEvictedPodCount: 0,
+			testFunc: func(podList []v1.Pod) {
+				for _, pod := range podList {
+					pod.DeletionTimestamp = &metav1.Time{}
+				}
+			},
 		},
 	}
 
 	for _, test := range tests {
 		// create fake client
 		fakeClient := &fake.Clientset{}
+		test.testFunc([]v1.Pod{*p9, *p10})
```

Comment:
this is calling `testFunc` on p9 and p10 for every test, even though they are only used in one test. in this case you should just define those pods with the settings you need

Reply:
Already remove `testFunc`, may I click `Resolve conversation`?